webapi/src/main/java/org/sakaiproject/webapi/controllers/RubricsRestController.java

```diff
@@ -370,7 +371,9 @@ public ResponseEntity<byte[]> getPdf(@PathVariable String siteId, @PathVariable
         checkSakaiSession();
 
         ContentDisposition contentDisposition = rubricsService.getRubric(rubricId).map(rubric -> {
-            return ContentDisposition.builder("attachment").filename(rubric.title + ".pdf").build();
+            String filename = StringUtils.trimToEmpty(rubric.title).replace(".", "_");
```

Comment:
This is good, but we should maybe block using periods when creating the rubric, also. I'm not suggesting do it here, just something to add in the future.

Reply:
Maybe. It's an edge case for sure, but: Can it break something? Is there no value in it?